src/firefly/java/edu/caltech/ipac/firefly/server/query/AlgorithmDefinition.java

```diff
@@ -0,0 +1,29 @@
+package edu.caltech.ipac.firefly.server.query;
```

Comment:
Please include on top of file the caltech/ipac license's information.  See other source file for example.